account_chart_update/model/account_tax_code.py

```diff
@@ -25,3 +25,13 @@ class AccountTaxCode(models.Model):
     _inherit = 'account.tax.code'
 
     active = fields.Boolean('Active', default=True)
+
+    def _sum(self, cr, uid, ids, name, args, context, where='',
+             where_params=()):
+        try:
+            return super(AccountTaxCode, self)._sum(
+                cr, uid, ids, name, args, context, where=where,
+                where_params=where_params)
+        except:
+            cr.rollback()
+            return dict.fromkeys(ids, 0.0)
```

Comment:
This is because this search return an empty value: https://github.com/OCA/OCB/blob/8.0/addons/account/account.py#L1708, and then it's used in an SQL query, provoking an error. It's a quick and a little dirty patch for that.

You can raise the problem browsing with advanced search inactive tax codes.